app/scripts/services/quota.js

```diff
@@ -277,6 +282,45 @@ angular.module("openshiftConsole")
         return _.some(quotas, isExceeded) || _.some(clusterQuotas, isExceeded);
     };
 
+    //Specific to storage quotas for max claim count and max storage usage
+    // and the amount of claims and storage already used
+
+    var getStorageQuotas = function(quotas) {
+      // Currently, there can be multiple quotas per namespace (bug?)
+      // So let's go through them and find the most restrictive
+      var maxNumberClaims = 0;
+      var maxStorage = 0;
+      var claimCountUsed = 0;
+      var claimStorageUsed = 0;
```

Comment:
Defaults should probably be undefined instead of 0 anyway so you can tell the difference between a zero value and unset.